src/ContentLoader.js

```diff
@@ -24,15 +40,20 @@ export default function CustomContentLoader({
 }
 
 CustomContentLoader.propTypes = {
+  id: PropTypes.string,
   children: PropTypes.oneOfType([
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]).isRequired,
-  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+  width: PropTypes.number,
+  height: PropTypes.number
 };
 
 CustomContentLoader.defaultProps = {
-  width: '100%',
-  height: '100%'
+  id:
+    Math.random()
+      .toString(36)
+      .substring(2) + Date.now().toString(36),
```

Comment:
We are generating these random values in a few places now, shouldn't we just include a dependency to help us with these? [shortid](https://github.com/dylang/shortid) or something similar?

Reply:
🤔